model/riscv_fdext_regs.sail

```diff
@@ -363,6 +456,6 @@ function accrue_fflags(flags) = {
   then {
     fcsr->FFLAGS() = f;
     update_softfloat_fflags(f);
-    dirty_fd_context();
+    if ~ (sys_enable_zfinx()) then dirty_fd_context();
```

Comment:
I think both are ok, I could go either way on it, though I think I prefer this, since there is no FD context on Zfinx and there should probably be an assert in dirty_fd_context that it's not Zfinx (haven't checked if this patch adds it or not).

Reply:
Rename the function to `dirty_fd_context_if_exists()` then? It's not great to keep repeating the conditional like this. Whether those bits exist or not should be defined in exactly one place.